src/fairseq2/models/wav2vec2/asr/factory.py

```diff
@@ -96,10 +120,49 @@ def _base_100h() -> Wav2Vec2AsrConfig:
     config = _base_10h()
 
     config.encoder_config = _base_100h_encoder()
-
     return config
 
 
+@wav2vec2_asr_arch("large_ls960_10h")
+def _large_ls960_10h() -> Wav2Vec2AsrConfig:
+    """wav2vec2 large arch trained on the Librispeech 960h dataset."""
+    return Wav2Vec2AsrConfig(
```

Comment:
For better readability, I prefer the convention:

```
config = _base_10h()

config.encoder_config = ...

return config
```

instead of `config = Wav2Vec2AsrConfig(...)`.